localization/pose_instability_detector/README.md

```diff
@@ -0,0 +1,46 @@
+# pose_instability_detector
+
+The `pose_instability_detector` package includes a node designed to monitor the stability of `/localization/kinematic_state`, which is an output topic of the Extended Kalman Filter (EKF).
+
+This node triggers periodic timer callbacks to compare two poses:
+
+- The pose obtained by integrating the twist values from the last received message on `/localization/kinematic_state` over a duration specified by `interval_sec`.
+- The latest pose from `/localization/kinematic_state`.
+
+The results of this comparison are then output to the `/diagnostics` topic.
+
+If this node outputs WARN messages to `/diagnostics`, it means that the EKF output is significantly different from the integrated twist values.
+This discrepancy suggests that there may be an issue with either the `pose_estimator` or the `twist_estimator`.
+
+The following diagram provides an overview of what the timeline of this process looks like:
+
+![timeline](./media/timeline.drawio.svg)
+
+## Parameters
+
+See [pose_instability_detector.param.yaml](config/pose_instability_detector.param.yaml).
+
+| Name                  | Type   | Description                         |
+| --------------------- | ------ | ----------------------------------- |
+| `interval_sec`        | double | Interval of timer callback (second) |
+| `threshold_linear_x`  | double | Threshold of linear x (m)           |
+| `threshold_linear_y`  | double | Threshold of linear y (m)           |
+| `threshold_linear_z`  | double | Threshold of linear z (m)           |
+| `threshold_angular_x` | double | Threshold of angular x (rad)        |
+| `threshold_angular_y` | double | Threshold of angular y (rad)        |
+| `threshold_angular_z` | double | Threshold of angular z (rad)        |
```

Comment:
I think it is better following [this guide using json schema](https://github.com/orgs/autowarefoundation/discussions/3371).
How do you think?

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/5439/commits/e06c66add3c1ccb6c46c7902032a24033fe5d686.

`mkdocs serve` in `autoware/src/universe/autoware.universe` and go to <http://127.0.0.1:8000/autoware.universe/localization/pose_instability_detector/>

I can see a following table.

![Screenshot from 2023-10-31 13-51-42](https://github.com/autowarefoundation/autoware.universe/assets/28504069/9149c32f-5816-4eb8-80b8-bfd77e16a4f9)


and the schema check is passed
https://github.com/autowarefoundation/autoware.universe/actions/runs/6702224534/job/18210896634?pr=5439